appserver/ejb/ejb-rest-remoting/admin/src/main/java/fish/payara/ejb/rest/admin/DisableEjbInvokerCommand.java

```diff
@@ -0,0 +1,115 @@
+ *
+ * GPL Classpath Exception:
+ * The Payara Foundation designates this particular file as subject to the "Classpath"
+ * exception as provided by the Payara Foundation in the GPL Version 2 section of the License
+ * file that accompanied this code.
+ *
+ * Modifications:
+ * If applicable, add the following below the License Header, with the fields
+ * enclosed by brackets [] replaced by your own identifying information:
+ * "Portions Copyright [year] [name of copyright owner]"
+ *
+ * Contributor(s):
+ * If you wish your version of this file to be governed by only the CDDL or
+ * only the GPL Version 2, indicate your decision by adding "[Contributor]
+ * elects to include this software in this distribution under the [CDDL or GPL
+ * Version 2] license."  If you don't indicate a single choice of license, a
+ * recipient has the option to distribute your version of this file under
+ * either the CDDL, the GPL Version 2 or to extend the choice of license to
+ * its licensees as provided above.  However, if you add GPL Version 2 code
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+ */
+package fish.payara.ejb.rest.admin;
+
+import static org.glassfish.config.support.CommandTarget.CLUSTER;
+import static org.glassfish.config.support.CommandTarget.CLUSTERED_INSTANCE;
+import static org.glassfish.config.support.CommandTarget.CONFIG;
+import static org.glassfish.config.support.CommandTarget.DAS;
+import static org.glassfish.config.support.CommandTarget.STANDALONE_INSTANCE;
+import static org.glassfish.deployment.autodeploy.AutoDeployer.getNameFromFilePath;
+
+import java.nio.file.Path;
+
+import javax.inject.Inject;
+
+import org.glassfish.api.ActionReport;
+import org.glassfish.api.admin.AdminCommand;
+import org.glassfish.api.admin.AdminCommandContext;
+import org.glassfish.api.admin.ExecuteOn;
+import org.glassfish.api.admin.RuntimeType;
+import org.glassfish.api.admin.ServerEnvironment;
+import org.glassfish.config.support.TargetType;
+import org.glassfish.deployment.autodeploy.AutoDeployer.AutodeploymentStatus;
+import org.glassfish.deployment.autodeploy.AutoUndeploymentOperation;
+import org.glassfish.hk2.api.PerLookup;
+import org.glassfish.hk2.api.ServiceLocator;
+import org.jvnet.hk2.annotations.Service;
+
+/**
+ * This command disables the EJB invoker endpoint.
+ * 
+ * <p>
+ * This happens by undeploying the small application in <code>/domains/[domain]/endpoints/ejb-invoker</code>.
+ * Note that by default this application is not deployed. The default context root of this application is 
+ * <code>/ejb-invoker</code>, but it may have been set to a different root by the {@link EnableEjbInvokerCommand}
+ * command.
+ * 
+ * @author Arjan Tijms
+ *
+ */
+@Service(name = "disable-ejb-invoker")
+@PerLookup
+@ExecuteOn(RuntimeType.DAS)
+@TargetType(value = { DAS, STANDALONE_INSTANCE, CLUSTER, CLUSTERED_INSTANCE, CONFIG })
```

Comment:
Is this applicable to deployment groups as well?

Reply:
Not really tbh

Reply:
Actually AFAIK this line can be deleted entirely, the command doesn't use the target utility

Reply:
In fact, should this command allow deploying the endpoint to instances other than the DAS? It currently doesn't deploy to anything other than the DAS

Reply:
After discussion, yes, it should target other instances as well. I've added that it the latest commits.